source/tutorials/test-strategy/test-strategy.rst

```diff
@@ -54,41 +54,20 @@ Android Validation Out-Scope
 * Pre-Certification
 * Documentation
 
-Validation Activates Key Flow
+Validation Activity Flow
 -----------------------------
 
 .. figure:: images/validation-activates-key-flow.png
     :align: center
 
-Test Suite
-----------
-
-The objective of the System Functional Test is to ensure 100% validation coverage
-for applicable features where required and evaluate end-to-end functionalities of
-the product.
-
-.. list-table::
-    :widths: 50 50
-    :header-rows: 1
-
-    * - Test Suite
-      - Description
-    * - System Functional
-      - | Requirement-based testing. Test cases are end-to-end and cover features
-        | captured in Jira (manual + automation)
-    * - PnP - Performance
-      - Measures performance KPIs
-    * - Google Compliance
-      - CiC and CiV architectures are designed to meet full CDD compliance
-
-Test Suite and Defect Tracking
+System Functional Testing and Defect Tracking
 ------------------------------
```

Comment:
The length of underlying characters must match the length of the section title.

Reply:
got it; will correct.